src/main/java/org/prebid/server/util/HttpUtil.java

```diff
@@ -150,4 +150,8 @@ public static String getDomainFromUrl(String url) {
         return context.cookies().stream()
                 .collect(Collectors.toMap(Cookie::getName, Cookie::getValue));
     }
+
+    public static String cookieToSetCookie(Cookie cookie) {
```

Comment:
Please rename to `cookieToHeaderValue`, this name is confusing

Reply:
or m.b. something like `toSetCookieHeaderValue`. What do you think guys?

Reply:
`resolveSetCookieHeaderValue`?